api/resolver/provider.go

```diff
@@ -1,17 +1,21 @@
-package api
+package resolver
 
 import (
 	"github.com/google/wire"
 	"github.com/robojones/graphql/api/resolver/mutation"
 	"github.com/robojones/graphql/api/resolver/mutation/auth"
+	projectMutation "github.com/robojones/graphql/api/resolver/mutation/project"
```

Comment:
That's kind of ugly.. I would have created a new wire set in the mutation folder, and then import that provider here.

Reply:
Didn't bother me but I fixed it in b1b543fabb21916257c96c2f4c58514e114b92da